orchestrator/modules/tools/discovery/platform_executor.py

```
@@ -0,0 +1,667 @@
+                    "success": True,
+                    "requires_confirmation": True,
+                    "action": action_name,
+                    "permission_level": action_def.permission_level,
+                    "message": (
+                        f"This action ({action_def.permission_level}) requires confirmation. "
+                        f"Action: {action_name} — {action_def.description[:100]}"
+                    ),
+                    "params": params,
+                }
+        except Exception:
+            pass  # If registry check fails, proceed (fail open for read actions)
```

Ruff (0.15.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)